package.json

```diff
@@ -5,7 +5,6 @@
     "abilliti-backend": "bin/abilliti-backend.js"
   },
   "scripts": {
-    "build": "tsc",
```

Comment:
Remove build command so it won't generate .js and .d.ts types which are not needed for deployment and just clutter the workspace.